.tekton/pull-request.yaml

```
@@ -44,6 +44,66 @@ spec:
         params:
         - name: args
           value: ["."]
+      - name: build-container
+        runAfter:
+          - yaml-lint-check
+        params:
+          - name: IMAGE
+            value: quay.io/redhat-appstudio/pull-request-builds:build-definitions-{{revision}}
+          - name: CONTEXT
+            value: appstudio-utils
+        taskRef:
+          name: buildah
+        workspaces:
+          - name: source
+            workspace: workspace
+      - name: build-bundles
+        params:
+          - name: revision
+            value: "{{ revision }}"
+        runAfter:
+          - build-container
+        workspaces:
+          - name: source
+            workspace: workspace
+        taskSpec:
+          params:
+            - name: revision
+              type: string
+          steps:
+            - name: build-bundles
+              image: quay.io/redhat-appstudio/pull-request-builds:build-definitions-{{revision}}
+              workingDir: $(workspaces.source.path)
+              script: |
+                #!/usr/bin/env bash
+                MY_QUAY_USER=redhat-appstudio TEST_REPO_NAME=pull-request-builds BUILD_TAG=$(params.revision) SKIP_BUILD=1 hack/build-and-push.sh
+          workspaces:
+            - name: source
+      - name: e2e-test
+        runAfter:
+          - build-bundles
+        taskSpec:
+          steps:
+            - name: e2e-test
+              image: quay.io/psturc/e2e-tests:PLNSRVCE-149
+              imagePullPolicy: Always
+              args: [
+                "--ginkgo.focus=has-suite",
+                "--ginkgo.progress",
+                "--ginkgo.v"
+              ]
+              env:
+              - name: GITHUB_TOKEN
+                value: "<token-for-pushing-app-and-component-to-github>"
+              - name: QUAY_E2E_ORGANIZATION
+                value: redhat-appstudio
+              - name: GITHUB_E2E_ORGANIZATION
+                value: redhat-appstudio
+              - name: HAS_BUILD_NAMESPACE
```

probably needs to be updated to
```suggestion
              - name: E2E_APPLICATIONS_NAMESPACE
```
but I'll update it once https://github.com/redhat-appstudio/e2e-tests/pull/47 is merged